sass/cd/_cd-variables.scss

```diff
@@ -75,16 +75,17 @@
   --cd-font: helvetica, arial, sans-serif;
   --cd-font--system: system-ui;
 
-  // Multilingual
-  // https://brand.unocha.org/d/xEPytAUjC3sH/visual-identity#/basics/fonts/multi-language-typefaces-1
-  --cd-font--noto: "Noto Sans", sans-serif;
-  --cd-font--noto-kufi: "Noto Kufi Arabic", sans-serif;
-
   // Roboto
   // https://brand.unocha.org/d/xEPytAUjC3sH/visual-identity#/basics/fonts/advanced-users
   --cd-font--roboto: "Roboto", helvetica, arial, sans-serif;
   --cd-font--roboto-condensed: "Roboto Condensed", helvetica, arial, sans-serif;
   --cd-font--roboto-slab: "Roboto Slab", helvetica, arial, sans-serif;
+
+  // Multilingual
+  // https://brand.unocha.org/d/xEPytAUjC3sH/visual-identity#/basics/fonts/multi-language-typefaces-1
+  --cd-font--noto: "Noto Sans", sans-serif;
```

Comment:
I would argue that the variable should be named `--cd-font--noto-sans` to better reflect the real font name (smae above with `noto-kufi-arabic` and `noto-sans-sc`, notably to be more explicit and differenciate between Noto Sans and Noto Serif for example.